### PR TITLE
SVGLoader: adding support for elliptical arc

### PR DESCRIPTION
**Pull request by ndebeiss**
#13478

### CODE REVIEW

**Comment by mrdoob** [comment]
Looking good!

File: examples/js/loaders/SVGLoader.js [hunk 1]
```diff
@@ -221,14 +221,13 @@ THREE.SVGLoader.prototype = {
 						break;
 
 					case 'A':
-						console.warn( command );
 						var numbers = parseFloats( data );
 						for ( var j = 0, jl = numbers.length; j < jl; j += 7 ) {
-							// TODO
-							point.x = numbers[ j + 5 ];
```
Comment (mrdoob) [inline examples/js/loaders/SVGLoader.js]: You shouldn't remove this. You'll break further relative commands otherwise.

Comment (ndebeiss) [reply examples/js/loaders/SVGLoader.js]: right, I corrected it in next commit

Comment (mrdoob) [reply examples/js/loaders/SVGLoader.js]: Thanks! I'll merge and clean up.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
Alright. Things should be a bit better now, you were passing the incorrect start position.
Still quite a bit more to clean up, the test files I'm using still don't display correctly.

**Comment by mrdoob** [comment]
I've gone through the original code and ported it more thoroughly. Seems like you left some bits. Things are starting to look better!

**Comment by mflux** [comment]
That doesn't look quite right: https://www.dropbox.com/s/9l3qwd61ioy0deo/Asset%201.svg?dl=0

screenshot 2018-05-18 18 49 21

SVG was exported from illustrator with the following settings (via export asset):

screenshot 2018-05-18 18 50 12

**Comment by mflux** [comment]
I discovered the issue. Adobe seems to think 

    "M.0035.5s72,.5,72,32"

Is valid SVG commands. You see M here only has one value for a coordinate and assumes the second value is zero.

By adding optional zero conditions in the loader, I'm able to fix this issue. Not sure if I sure do a PR on this or easy enough fix by someone who owns the loader.

							point.x = numbers[ j + 0 ] || 0;
							point.y = numbers[ j + 1 ] || 0;

**Comment by mrdoob** [comment]
> I discovered the issue. Adobe seems to think 
> 
>     "M.0035.5s72,.5,72,32"
> 
> Is valid SVG commands. You see M here only has one value for a coordinate and assumes the second value is zero.

That's not the command that is broken in that svg but the issue is the same. The broken command is:

```
C2.6241,38.9467-1.6825,48.6037.71,53.7206
```

screen shot 2018-05-21 at 4 41 39 pm

**Comment by mrdoob** [comment]
@mflux the latest loader should be able to handle these files now...

**Pull request merged**